modules/rosa-how-upgrades-work.adoc

```diff
@@ -0,0 +1,56 @@
+
+// Module included in the following assemblies:
+//
+// upgrading/rosa-upgrading-sts.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="rosa-how-upgrades-work_{context}"]
+= How ROSA (classic architecture) cluster upgrades work
+
+Upgrades are manually initiated (one-time) or automatically scheduled (recurring). Red{nbsp}Hat Site Reliability Engineers (SREs) monitor upgrade progress and either proactively notify you to take corrective actions or remedy issues encountered.
+
+The Cluster Version Operator (CVO) is the primary component that orchestrates and facilitates the OpenShift Container Platform update process. The scheduling, monitoring, and notifications of ROSA (classic architecture) cluster upgrades are orchestrated through the Managed Upgrade Operator (MUO). The MUO manages the orchestration of automated in-place cluster upgrades by satisfying the operating conditions that a managed cluster must hold before and after initiating the cluster upgrade.
+
+[id="rosa-upgrade-scheduled-time_{context}"]
+== Cluster upgrade scheduled time
+
+You can schedule cluster upgrades by setting the scheduled time. This is when the preparation for the cluster upgrade begins with pre-upgrade health checks and additional compute capacity creation. The actual cluster upgrade starts within one hour from the scheduled time. You receive an email notification when the cluster upgrade starts.
+
+The Pre-Health Check (PHC) provides extra protection to ensure the scheduled update proceeds as expected and runs in the following two scenarios:
+
+* If an upgrade is scheduled for more than 2 hours from the current time, the PHC runs and the user is notified if there are any failures. This PHC is in the _New_ phase of the upgrade.
+* When the upgrade is immediate or within 2 hours, the PHC runs just before the upgrade begins. This PHC is in the _Upgrading_ phase of the upgrade.
+This means that PHC is always run at least one time during the upgrading phase but can also be run additionally in advance if the upgrade is scheduled for more than 2 hours from the current time.
+
+You can observe the status of the cluster upgrade by running the `oc get clusterversion` command in the ROSA CLI (`rosa`).
```

Comment:
It would be better to use 'rosa describe upgrade --cluster=<cluster name|cluster_id>' instead of 'oc get clusterversion'